<ui version="4.0" >
 <class>UiNewEditDocumentBase</class>
 <widget class="QWidget" name="UiNewEditDocumentBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>New / Edit Document</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../../resource/dms.qrc" >:/picture/16/images/16x16/documents_16.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="2" >
    <widget class="QTreeWidget" name="treeWidgetDocument" >
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <property name="sortingEnabled" >
      <bool>true</bool>
     </property>
     <column>
      <property name="text" >
       <string>User</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Document</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Group</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Documentpath</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="labelDocumentId" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>Document ID:</string>
       </property>
       <property name="buddy" >
        <cstring>lineEditDocumentId</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditDocumentId" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="frame" >
        <bool>true</bool>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDocId" >
       <property name="minimumSize" >
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="../../resource/dms.qrc" >:/picture/16/images/16x16/reload_16.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="labelUser" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>User:</string>
       </property>
       <property name="buddy" >
        <cstring>comboBoxUser</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxUser" />
     </item>
     <item>
      <widget class="QPushButton" name="btnUser" >
       <property name="minimumSize" >
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item rowspan="3" row="2" column="1" >
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QPushButton" name="btnAddDocument" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnUpdateDocument" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDeleteDocument" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="labelUser_2" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>Group:</string>
       </property>
       <property name="buddy" >
        <cstring>comboBoxGroup</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxGroup" />
     </item>
     <item>
      <widget class="QPushButton" name="btnGroup" >
       <property name="minimumSize" >
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="labelDocumentName" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>Document name:</string>
       </property>
       <property name="buddy" >
        <cstring>lineEditDocumentName</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditDocumentName" >
       <property name="frame" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="labelDocument" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>Document:</string>
       </property>
       <property name="buddy" >
        <cstring>lineEditDocumentPath</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditDocumentPath" >
       <property name="frame" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSelectDocument" >
       <property name="minimumSize" >
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="2" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="btnOk" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>111</width>
         <height>26</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>treeWidgetDocument</tabstop>
  <tabstop>lineEditDocumentId</tabstop>
  <tabstop>btnDocId</tabstop>
  <tabstop>comboBoxUser</tabstop>
  <tabstop>comboBoxGroup</tabstop>
  <tabstop>lineEditDocumentName</tabstop>
  <tabstop>lineEditDocumentPath</tabstop>
  <tabstop>btnSelectDocument</tabstop>
  <tabstop>btnAddDocument</tabstop>
  <tabstop>btnUpdateDocument</tabstop>
  <tabstop>btnDeleteDocument</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources>
  <include location="../../resource/dms.qrc" />
 </resources>
 <connections/>
</ui>
